Python test debug

corepointer opened the pull request:
creating a PR for this to check if all tests work now (tests triggered by PR to master)
will merge myself

corepointer committed:
debugging python tests & workflow action (very messy :-P)

corepointer committed:
uncommented the other py tests for more checking

corepointer committed:
silenced warnings in more tests; shutdown gateway manually

corepointer committed:
proc.communicate; unit test dont exit; sleep and check return

kev-inn commented:
I would prefer some other way to shutdown the server than `shutdown`. Fist of all a check is missing since we only start a subprocess if no SystemDS instance is running yet, secondly if we need a explicit shutdown it makes sense to also have an explicit start. As I have already suggested in my PR it would make sense to use a ContextManager (See https://book.pythontips.com/en/latest/context_managers.html). We should make sure that we are happy with our decision since it will likely stay for longer, I hoped that we find a way to automatically shutdown the java server process on exit of the python program, but a ContextManager would also be a good idea.

corepointer committed:
cleanup

# Conflicts:
#	.github/disabled-workflows/workflows/python.yml
#	src/main/python/tests/test_l2svm.py

corepointer committed:
cleanup + resolving rebase conflicts

# Conflicts:
#	.github/disabled-workflows/workflows/python.yml
#	src/main/python/tests/test_l2svm.py

corepointer committed:
two more fixes for pylintest:
* ignore warning and shutdown gateway for lineage tests
* change encoding of lineage test files to utf8 (for the reference: iconv --from-code=ISO-8859-1 --to-code=UTF-8 in.txt > out.txt)

corepointer committed:
removed disabled workflow; added a new line at eof
